person-service/src/main/kotlin/com/thehuginn/controller/dto/response/PersonResponse.kt

```diff
@@ -0,0 +1,13 @@
+package com.thehuginn.controller.dto.response
+
+import com.thehuginn.enums.PersonRole
+import java.util.UUID
+
+data class PersonResponse (
+    val id: UUID,
+    val firstName: String,
+    val lastName: String,
+    val email: String,
+    val role: PersonRole,
```

Comment:
val type: PersonType mi pride vhodnejsie

Reply:
Pouzivam ptm kludne bud klavesove skratky, alebo right click  -> refactor -> rename alebo tak nejak. To iste aj pre premenne